node/core/pvf/src/executor_intf.rs

```diff
@@ -278,11 +278,21 @@ impl TaskExecutor {
 }
 
 impl sp_core::traits::SpawnNamed for TaskExecutor {
-	fn spawn_blocking(&self, _: &'static str, future: futures::future::BoxFuture<'static, ()>) {
+	fn spawn_blocking(
+		&self,
+		_: &'static str,
+		_: &'static str,
```

Comment:
nit: even if we don't use either, it'd be nice to use `_subsystem` and `_task_name` instead of two `_`